packages/lib/jobs/definitions/internal/seal-document.ts

```diff
@@ -141,7 +152,9 @@ export const SEAL_DOCUMENT_JOB_DEFINITION = {
       }
 
       for (const field of fields) {
-        await insertFieldInPDF(pdfDoc, field);
+        if (field.inserted) {
```

Comment:
question: should we remove the uninserted (optional) fields from the documents altogether? at the moment, they're not inserted in the PDF, but they are still existing in the db and are linked to the document.

Reply:
they should remain